esutil/bulk_indexer.go

```diff
@@ -408,10 +409,19 @@ func (w *worker) writeMeta(item BulkIndexerItem) error {
 		w.buf.Write(w.aux)
 		w.aux = w.aux[:0]
 	}
-	if item.Index != "" {
+	if item.Routing != "" {
 		if item.DocumentID != "" {
 			w.buf.WriteRune(',')
 		}
+		w.buf.WriteString(`"_routing":`)
```

Comment:
In my testing this needed to be replaced with 
```suggestion
		w.buf.WriteString(`"routing":`)
```

Reply:
Can this change be taken and the pr accepted?